### PR TITLE
Add aliases command

### PR DESCRIPTION
**Pull request by knownasilya**
`aliases(['i', 'init'])` or `aliases('i')`

Also can be just `alias`, if you prefer.

### CODE REVIEW

File: lib/command.js [hunk 1]
```diff
@@ -132,6 +133,19 @@ Command.prototype.parseExpectedArgs = function(args){
...
 };
 
+Command.prototype.aliases = function (names) {
+  var self = this;
+
+  if (!Array.isArray(names)) {
+    this._aliases.push(names);
+  }
+  else {
+    this._aliases = names;
```
Comment (calvinmetcalf) [inline lib/command.js]: this._aliases = this._aliases.concat(names);

File: lib/command.js [hunk 2]
```diff
@@ -132,6 +133,19 @@ Command.prototype.parseExpectedArgs = function(args){
   return this;
 };
 
+Command.prototype.aliases = function (names) {
+  var self = this;
```
Comment (calvinmetcalf) [inline lib/command.js]: is this ever used?

File: lib/command.js [hunk 3]
```diff
@@ -132,6 +133,19 @@ Command.prototype.parseExpectedArgs = function(args){
   return this;
 };
 
+Command.prototype.aliases = function (names) {
```
Comment (calvinmetcalf) [inline lib/command.js]: can we have alias as an alias to aliases? aka 

`Command.prototype.aliases = Command.prototype.alias = function (names) {`

File: lib/command.js (hunk 2 above)
Comment (knownasilya) [inline lib/command.js]: ops, forgot to remove

File: lib/command.js (hunk 3 above)
Comment (knownasilya) [inline lib/command.js]: yes, good idea, should we still allow a string for `aliases`?

**Comment by knownasilya** [comment]
Is this better?

**Pull request merged**